src/System.Net.Http.WinHttpHandler/src/project.json

```diff
@@ -2,7 +2,9 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-23908",
     "Microsoft.Win32.Primitives": "4.0.0",
+    "System.Collections": "4.0.0",
     "System.Diagnostics.DiagnosticSource": "4.0.0-rc3-23908",
+    "System.Diagnostics.Debug": "4.0.10",
```

Comment:
I don't quite follow why a couple extra random packages popped up... but I assume it was to stabilize the version they were using across the dnxcore to netstandard switch?